### PR TITLE
Add UWP as platform that supports .NET Standard

### PR DESCRIPTION
**Pull request by philippgille**
https://blogs.msdn.microsoft.com/dotnet/2017/10/10/announcing-uwp-support-for-net-standard-2-0/

### CODE REVIEW

**Comment by philippgille** [comment]
Well, the "Fall Creators Update" constraint isn't even necessary. UWP 10.0 supports .NET Standard 1.0, as seen in [this table](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support). What do you think?

**Review by jzeferino — COMMENTED** [review]
LGTM. 
Also I recommended to add more platforms see #3814.

File: docs/standard/class-libraries.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ The .NET Standard does not expose the entire functionality of the .NET Framework
 The following platforms support .NET Standard libraries:
 
 *   .NET Core
+*   Universal Windows Platform (UWP), Fall Creators Update or newer
```
Comment (mairaw) [inline docs/standard/class-libraries.md]: should we simply list the version here instead?
Universal Windows Platform (UWP) 10.0 and later versions

@terrajobst?

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @philippgille. Please see my comment regarding how I think we should list UWP here.

**Comment by philippgille** [comment]
@mairaw: Thanks for the advice. I also thought about @jzeferino's advice. I decided that instead of listing all the platforms with their versions, where the info which .NET Standard version is supported is still missing, a listing with just the platform names *plus* a link to the [.NET Standard versions / .NET platforms versions matrix](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support) might be best. What do you think?

Also: Should I squash the commits?

**Comment by philippgille** [comment]
@mairaw: Please see [my previous comment](https://github.com/dotnet/docs/pull/3813#issuecomment-348692601)

**Review by mairaw — CHANGES_REQUESTED** [review]
Apologies on not getting back to you sooner @philippgille . I lost track of this PR. I left a small comment for you but changes overall look good. Let me know if you'd like me to address the feedback for you.

**Review by mairaw — APPROVED** [review]

**Pull request merged**